LibraryBooks.Common.Entities/User.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace LibraryBooks.Entities
+{
+    public class User
+    {
+
+        public int UserId { get; private set; }
+
+        public string SurName { get; private set; }
+
+        public string FirstName { get; private set; }
+
+        public string SecondName { get; private set; }
```

Comment:
`Surname` *is* a `Family name` or a `Second name`, or `Last name`. You probably meant to have a `Patronymic` or a `Middle name`. Also - not all countries have middlename, so it should be optional